src/app.py

```diff
@@ -20,9 +20,12 @@
 def get_prompt():
 	""" Build a prompt based on the current logged in user or guest """
 	username = 'guest'
+	working = ''
 	if current_user.is_authenticated:
 		username = current_user.username
-	return f'{username}@{request.host} $ '
+		#how to call the click command for working here instead of writing the query again?
+		working = models.Directory.get(models.Directory.id == current_user.working_directory).get_full_path()
```

Comment:
Store and get working from the session (check out flask-login documentation) so that non logged in users can cd and ls around.

Reply:
Done.